docs/make.jl

```diff
@@ -97,6 +97,7 @@ makedocs(
         ],
         "References" => "references.md",
         "API" => "api.md",
+        "Reproducibility of Breeze.jl models" => "reproducibility.md",
```

Comment:
this could also go in "appendix", I am fine either way

Reply:
Done!